DuckDuckGo/TabBar/ViewModel/TabCollectionViewModel.swift

```diff
@@ -153,6 +154,32 @@ final class TabCollectionViewModel: NSObject {
                   burnerMode: burnerMode)
     }
 
+    var selectedTabCancellable: AnyCancellable?
+    private func subscribeToSelectedTab() {
```

Comment:
Out of curiosity, how come we send the pixel again when the user switch tabs? Won’t this generate noise in the pixels since it will send failures to load for the same webpage even tho it has not loaded multiple times?

Reply:
Future suggestion: Perhaps we can inject the fire function and unit test the behaviour

Reply:
Thanks, the requirement is to fire the pixel when the user sees the error screen and I checked this with Bobby as it would have been harder to have some state to only fire it the first time the error screen is seen and reset it, etc.